spring/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>spring</artifactId>
+    <parent>
+        <artifactId>hazelcast-jet-code-samples</artifactId>
+        <groupId>com.hazelcast.jet.samples</groupId>
+        <version>0.6-SNAPSHOT</version>
+    </parent>
+
+    <version>0.6-SNAPSHOT</version>
+
+
+    <properties>
+        <!-- needed for checkstyle/findbugs -->
+        <main.basedir>${project.parent.basedir}</main.basedir>
+
+        <spring.version>4.2.3.RELEASE</spring.version>
+        <hazelcast.spring.version>3.10-SNAPSHOT</hazelcast.spring.version>
```

Comment:
will this only work with a specific version of Hazelcast? We should add a note here.

Reply:
I've tried with 3.9 and below versions, 3.9 runs fine but below that we are getting class/method not found exceptions. So yes, user should use specific version of hazelcast, preferably the same version which jet shades.